src/services/session-manager.ts

```diff
@@ -103,7 +112,6 @@ export class OnlineSessionManager implements SessionManager {
           return;
         }
       }
-      this.updateSession({ status: 'non-authenticated' });
```

Comment:
What happens if the timeout occurs in the first request (when the status is _unknown_ and a spinner is being shown)?

Will be the spinner visible up to the next call to _GetUserData_?

Reply:
When status is unknown, account is empty, there is no user logged, so session value would be 'non-authenticated'
We only do nothing if the connection timed out and a user is logged in. 
In the first case, status goes from 'unknown' to 'non-authenticated' when timeout fails.
![image](https://user-images.githubusercontent.com/2439363/107067438-0b8a6b00-67be-11eb-9f77-7c1853804457.png)

I don´t see a way that can happen that user account is set, and it's the first time it's called. First time account shouldn´t be logged.

Reply:
Great explanation! I understand now, thanks!